docs/pages/get-started/installation.mdx

```diff
@@ -31,7 +31,7 @@ For Windows users we recommend using [PowerShell](https://docs.microsoft.com/en-
 
 ### Using Expo CLI
 
-You can use Expo CLI without installation by leveraging `npx` &mdash; a Node.js package runner.
+Inside a project, you can use Expo CLI by leveraging `npx` &mdash; a Node.js package runner. It is a part of the `expo` package and no installation is required to use it.
```

Comment:
```suggestion
Inside a project, you can use Expo CLI by leveraging `npx` &mdash; a Node.js package runner. It is a part of `npm` CLI and no installation is required to use it.
```

`npx` is a part of npm CLI, not Expo. In the past it was just a stand-alone executable packed, but then it got imported into `npm`:
* https://github.com/npm/npx

Reply:
Oh, I see the problem, I misunderstood to what exactly the second sentence is referring to (not `npx`, but Expo CLI). Maybe we can change "it" to be more descriptive, like "The CLI"?

But generally, in that case, we need to tweak the wording a bit anyway, since `expo` CLI is just an executable part of the package run via `npx` and it would work also outside expo project directory too (but in most cases will throw an error due to missing files).

Reply:
I went ahead and changed the wording. Thanks for pointing out the "it" part; I understand that it was confusing.